.github/workflows/perf-regression-ci.yml

```diff
@@ -145,7 +167,10 @@ jobs:
           export RESULT_REPO_BRANCH=${{ env.RESULT_REPO_BRANCH }}
           export RESULT_REPO_ACCESS_TOKEN=${{ secrets.CI_ACCESS_TOKEN }}
           export FROM_DATE=2020-07-10
-          ./ci-perf-kit/scripts/openjdk-history-run.sh ./mmtk-openjdk ./reports/${{ steps.branch.outputs.branch_name }}
+          ./ci-perf-kit/scripts/openjdk-history-run.sh \
```

Comment:
It might be easier to implement if we do two separate runs, one for the commit, one for the canary version, and then plot them in two separate graphs. In that case, the code for plotting the original graph does not need big changes, and can be used to plot the canary graph.

Reply:
OK I saw https://github.com/mmtk/ci-perf-kit/pull/47. You are doing that in `ci-perf-kit`. That's fine.

Reply:
@qinsoon The main concern is that `./ci-perf-kit/scripts/openjdk-history-run.sh` also commits to the `mmtk/ci-perf-result` repo and generates the HTML.  It is safer to use one single bash script to test both the newest version and the canary.